packages/server/postgres/migrations/2025-03-06T12:25:07.205Z_add-isPublic.ts

```diff
@@ -0,0 +1,24 @@
+import type {Kysely} from 'kysely'
+
+export async function up(db: Kysely<any>): Promise<void> {
+  await db.schema
+    .alterTable('Team')
+    .addColumn('isPublic', 'boolean', (col) => col.defaultTo(false).notNull())
+    .execute()
+
+  const featureFlags = await db
+    .selectFrom('FeatureFlag')
+    .innerJoin('FeatureFlagOwner', 'FeatureFlag.id', 'FeatureFlagOwner.featureFlagId')
+    .where('FeatureFlag.featureName', '=', 'publicTeams')
+    .select('FeatureFlagOwner.orgId')
+    .execute()
+
+  const orgIds = featureFlags.map((flag) => flag.orgId)
+  if (orgIds.length > 0) {
+    await db.updateTable('Team').set({isPublic: true}).where('orgId', 'in', orgIds).execute()
+  }
```

Comment:
+1 let's keep this in postgres if we can. I think you could just simplify this to
```ts
  const featureFlags = db
    .selectFrom('FeatureFlag')
    .innerJoin('FeatureFlagOwner', 'FeatureFlag.id', 'FeatureFlagOwner.featureFlagId')
    .where('FeatureFlag.featureName', '=', 'publicTeams')
    .select('FeatureFlagOwner.orgId')

  await db.updateTable('Team').set({isPublic: true}).where('orgId', 'in', featureFlags).execute()
```
and just omit the JS part.